src/vivarium_gates_lsff/constants/models.py

```diff
@@ -28,10 +28,11 @@ def __new__(cls, value):
 MEASLES_MODEL_NAME = data_keys.MEASLES.name
 MEASLES_SUSCEPTIBLE_STATE_NAME = f'susceptible_to_{MEASLES_MODEL_NAME}'
 MEASLES_WITH_CONDITION_STATE_NAME = MEASLES_MODEL_NAME
+MEASLES_RECOVERED_STATE_NAME = f'recovered_from_{MEASLES_MODEL_NAME}'
 MEASLES_MODEL_STATES = (MEASLES_SUSCEPTIBLE_STATE_NAME, MEASLES_WITH_CONDITION_STATE_NAME)
 MEASLES_MODEL_TRANSITIONS = (
     TransitionString(f'{MEASLES_SUSCEPTIBLE_STATE_NAME}_TO_{MEASLES_WITH_CONDITION_STATE_NAME}'),
-    TransitionString(f'{MEASLES_WITH_CONDITION_STATE_NAME}_TO_{MEASLES_SUSCEPTIBLE_STATE_NAME}'),
+    TransitionString(f'{MEASLES_WITH_CONDITION_STATE_NAME}_TO_{MEASLES_RECOVERED_STATE_NAME}'),
```

Comment:
Is this the result of measles not being something that can have reinfection?

Reply:
Correct - lifelong immunity assuming you recover.